terraform/globals.tf

```diff
@@ -104,3 +104,17 @@ variable "assets_prefix" {
   description = "Prefix for global assests like S3 buckets"
   default     = "gds-paas"
 }
+
+variable "log_groups_to_ship_to_csls" {
+  description = "The names of the log groups to ship to CSLS, without the -env suffix"
+  type        = "list"
+
+  default = [
+    "bosh_d_audit",
+    "bosh_d_audit_worker",
+    "bosh_d_auth",
+    "bosh_d_credhub_events",
```

Comment:
This is named `bosh_d_credhub_security_events_` elsewhere.

Reply:
Good catch

Force pushed resolved: https://github.com/alphagov/paas-bootstrap/compare/98f70903da4d8835dd0e38ed8ebf3495428bc474..5d36376c7cf9d63d5594741230ed3fb87c5233cd